src/traits.rs

```diff
@@ -774,10 +774,40 @@ pub trait SystemExt: Sized + Debug + Default + Send + Sync {
     /// use sysinfo::{System, SystemExt};
     ///
     /// let mut s = System::new_all();
+    /// s.refresh_cpu_usage();
+    /// ```
+    fn refresh_cpu_usage(&mut self) {
```

Comment:
Needs to update the first sentence of this function too: `Refreshes CPUs usage.`